addons/spike/functions/fnc_camera_setViewMode.sqf

```diff
@@ -29,11 +30,17 @@ if (_thermalMode > 1) then {
     if (_userInCamera) then {
         true setCamUseTI (_thermalMode - 2);
     };
-};
-if (_thermalMode == 1) then {
-    if (_userInCamera) then {
-        camUseNVG true;
+} else {
+    if (_thermalMode == 1) then {
+        if (_userInCamera) then {
+            camUseNVG true;
+        };
+    } else {
+        GVAR(ppEffectBW) ppEffectEnable true;
```

Comment:
add a new mode called `b&w`. this camera script is meant to be pulled out at somepoint to allow for MITL seekers of different types. No hardcoding for spike allowed